backend/src/services/aws/storage/storage.service.ts

```diff
@@ -0,0 +1,109 @@
+import {
+  CreateBucketCommand,
+  CreateBucketCommandOutput,
+  DeleteBucketCommand,
+  DeleteBucketCommandOutput,
+  DeleteObjectCommand,
+  DeleteObjectCommandOutput,
+  GetObjectCommand,
+  GetObjectCommandOutput,
+  PutObjectCommand,
+  PutObjectCommandOutput,
+  S3Client,
+  S3ServiceException,
+} from '@aws-sdk/client-s3';
+
+import {
+  StorageManipulateFileRequestDto,
+  StorageUploadFileRequestDto,
+} from '~/common/types/types';
+import { StorageError } from '~/exceptions/exceptions';
+
+type Constructor = {
+  region: string;
+  credentials: {
+    accessKeyId: string;
+    secretAccessKey: string;
+  };
+};
+
+class Storage {
```

Comment:
```suggestion
class File {
```
don't forget about file name